Styles/styles.css

```diff
@@ -0,0 +1,410 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300..800;1,300..800&family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap');
+/*
+Paars #5722CD;
+Blauw #4845E4;
+Roze #ED4385;
+Zwart #0C0C0C (voor standaard tekst);
+Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services") Uitzonderingen: Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)
+Grijs #7A7A7A (voor de service-tegel tekst);*/
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+.triangle-background {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 100%;
+    margin-top: -50px;
+    border-bottom: 50px solid white;
+    border-right: 1500px solid transparent;
+}
+
+.triangle-background-2 {
+    display: flex;
+    flex-direction: row;
+    margin: -250px auto 200px auto;
+    border-bottom: 50px solid transparent;
+    border-right: 1480px solid white;
+}
+
+.triangle-background-3 {
+    width: 100%;
+    border-bottom: 50px solid white;
+    border-right: 1500px solid transparent;
+}
+
+.triangle-background-4 {
+    width: 100%;
+    border-bottom: 50px solid transparent;
+    border-right: 1500px solid white;
+}
+    /* ------------------------------- 2. Typography ----------------------------------- */
+
+    h1 {
+        font-family: "Merriweather", serif;
+        font-weight: 700;
+        margin: 30px;
+        font-size: 44px;
+    }
+
+    h2 {
+        font-family: "Merriweather", serif;
+        font-weight: 700;
+        font-size: 32px;
+        margin: 30px 0;
+    }
+
+    h3 {
+        font-family: "Merriweather", serif;
+        font-size: 27px;
+        font-weight: 300;
+        color: white;
+    }
+
+    body {
+        font-family: "Roboto", serif;
+        font-weight: 200;
+        font-style: normal;
+        margin: 0 auto;
+    }
+
+    main {
+        max-width: 1600px;
+        color: black;
+    }
+
+    footer {
+        background-color: black;
+        color: white;
+        height: 70px;
+        max-width: 1500px;
+        border-bottom-left-radius: 5px;
+        border-bottom-right-radius: 5px;
+        margin: 0 auto 40px auto;
+    }
+    /* ------------------------------- 3. Layout --------------------------------------- */
+
+    /* --------------- 3.1 Reusables ----------------------- */
+
+    /* --------------- 3.2 Other --------------------------- */
+
+    .form-section {
```

Comment:
Het is handiger om de volgorde aan te laten sluiten op de volgorde in jouw HTML file. Dus begin met de header en werk vervolgens door tot de footer.